composables/getters/facetGetters.ts

```diff
@@ -22,7 +46,14 @@ const getBreadcrumbs = (searchData: {
   return [...homeCrumb, ...categoryCrumbs]
 }
 
+const getGrouped = (searchData: FacetResultsData, criteria?: string[]) => {
```

Comment:
Instead of creating new type (FacetResultsData), can we use any existing GraphQL type? If not then, give more meaningful name like searchDataParams instead of FacetResultsData?